src/sensai/evaluation/eval_util.py

```diff
@@ -49,29 +49,29 @@ def _isRegression(model: Optional[VectorModel], isRegression: Optional[bool]) ->
 
 
 def createVectorModelEvaluator(data: InputOutputData, model: VectorModel = None,
-        isRegression: bool = None, **kwargs) \
+                               isRegression: bool = None, **kwargs) \
```

Comment:
We do not use this identation style in sensAI, because
* it can give you _significantly_ less space in every new line than a double indent would, and gaining space is the very reason why we are breaking the line in the first place
* it is not robust against renaming (there are already a few places in the code base where you used this indentation style and where it has been broken due to renaming)
* it has no actual advantages over the double indent (yet the two aforementioned downsides).

I didn't tell you to change this in code you added (because it's not a big deal, though you should consider adjusting eventually ;) ), but I must object when you change existing lines.

Reply:
Pycharm did that for me when I changed the name of InputOutputData, I never touched those lines :(. Now that I fixed the naming with a fixup/rebase, these spaces remained. It's really annoying...

Reply:
Fixed it

Reply:
You can configure PyCharm not to do this:
![image](https://user-images.githubusercontent.com/1295099/106663411-fa661200-65a3-11eb-81f4-48ce6c6e9ec4.png)

One of these days, I will add some config files to the repo that all devs should import - for inspections and code style.

Reply:
Rebase is preferred to keep the history clean.
You can merge.

Reply:
Don't forget to send me your approval for the blessing of 1k new lines of code on develop ;)

Reply:
Alternatively, if you think the individual commits are not really needed, you could squash-merge, which should also fix the history.

Reply:
Btw, I have the same settings as in your screenshot

Reply:
Ah, then it must be these:
![image](https://user-images.githubusercontent.com/1295099/106664296-27ff8b00-65a5-11eb-9321-ce26bc6630d4.png)

Reply:
For once, it was good for github to have forced me to switch to the diff in order to approve, because it showed me that the indentation wasn't fixed. :)

Reply:
BTW (in case you weren't aware), the best way to use an interactive rebase that repositions the commit that fixes the indentation and squash it with the preceding commit.

Reply:
Still the same settings as mine :D